### PR TITLE
Stats pages enhancements

### PR DESCRIPTION
**Pull request by aleksandra-midor**

### CODE REVIEW

**Comment by aleksandra-midor** [comment]
[Answers.](https://trello.com/c/MIPAY9bZ/153-answers)

**Comment by aleksandra-midor** [comment]
[Answer 1](https://trello.com/c/uQAi6rKa/154-answer-1)

**Comment by aleksandra-midor** [comment]
[Answer 2](https://trello.com/c/oe2ZyYA4/155-answer-2)

**Comment by aleksandra-midor** [comment]
[insight window](https://trello.com/c/zn7eSm7v/158-insight-window)

File: src/pages/CreatorReasonsStats/index.js [hunk 1]
```diff
@@ -89,8 +90,8 @@ function CreatorReasonsStats(props) {
             people: answer.total,
           };
         })
-        .sort((a, b) => a.value - b.value)
-        .reverse();
+        .sort((a, b) => a.value - b.value);
+      // .reverse();
```
Comment (meerajm) [inline src/pages/CreatorReasonsStats/index.js:94]: Can we remove this comment if not required?

Comment (aleksandra-midor) [reply src/pages/CreatorReasonsStats/index.js:94]: ok

**Review by kosmolet — APPROVED** [review]

**Review by meerajm — APPROVED** [review]

**Pull request merged**